### ISSUE TITLE
possibly parameter MAX_FILE_UPLOAD_SIZE not working

### ISSUE THREAD

**Issue by rlagutinhub** [zulip/docker-zulip#197]
Please check, possibly parameter MAX_FILE_UPLOAD_SIZE not working
I set new value SETTING_MAX_FILE_UPLOAD_SIZE: '50' via docker-compose file, but allowed as before max upload 25MB

![123](https://user-images.githubusercontent.com/30324030/57178706-548ced00-6e7d-11e9-8e03-905afcf8d417.png)

```
[root@chat ~]#docker exec -it zulip bash
root@zulip:/#
root@zulip:/#
root@zulip:/#
root@zulip:/# cat /etc/zulip/settings.py | grep -v -e '^#' -e '^$'
from typing import Optional
ZULIP_ADMINISTRATOR = 'zulip-admin@example.com'
EXTERNAL_HOST = 'zulip.example.com'
AUTHENTICATION_BACKENDS = (
    'zproject.backends.EmailAuthBackend',  # Email and password; just requires SMTP setup
    # 'zproject.backends.GoogleMobileOauth2Backend',  # Google Apps, setup below
    # 'zproject.backends.GitHubAuthBackend',  # GitHub auth, setup below
    # 'zproject.backends.AzureADAuthBackend',  # Microsoft Azure Active Directory auth, setup below
    # 'zproject.backends.ZulipLDAPAuthBackend',  # LDAP, setup below
    # 'zproject.backends.ZulipRemoteUserBackend',  # Local SSO, setup docs on readthedocs
)
SSO_APPEND_DOMAIN = None  # type: Optional[str]
SESSION_EXPIRE_AT_BROWSER_CLOSE = False
SESSION_COOKIE_AGE = 60 * 60 * 24 * 7 * 2  # 2 weeks
ENABLE_FEEDBACK = False
FEEDBACK_EMAIL = ZULIP_ADMINISTRATOR
ENABLE_FILE_LINKS = False
LOCAL_UPLOADS_DIR = "/home/zulip/uploads"
MAX_FILE_UPLOAD_SIZE = 25
NAME_CHANGES_DISABLED = False
ENABLE_GRAVATAR = True
EMAIL_GATEWAY_PATTERN = ""
EMAIL_GATEWAY_LOGIN = ""
EMAIL_GATEWAY_IMAP_SERVER = ""
EMAIL_GATEWAY_IMAP_PORT = 993
EMAIL_GATEWAY_IMAP_FOLDER = "INBOX"
import ldap
from django_auth_ldap.config import LDAPSearch
AUTH_LDAP_SERVER_URI = ""
AUTH_LDAP_BIND_DN = ""
AUTH_LDAP_USER_SEARCH = LDAPSearch("ou=users,dc=example,dc=com",
                                   ldap.SCOPE_SUBTREE, "(uid=%(user)s)")
LDAP_APPEND_DOMAIN = None  # type: Optional[str]
LDAP_EMAIL_ATTR = None  # type: Optional[str]
AUTH_LDAP_USER_ATTR_MAP = {
    # full_name is required; common values include "cn" or "displayName".
    # If names are encoded in your LDAP directory as first and last
    # name, you can instead specify first_name and last_name, and
    # Zulip will combine those to construct a full_name automatically.
    "full_name": "cn",
    # "first_name": "fn",
    # "last_name": "ln",
    # User avatars can be pulled from the LDAP "thumbnailPhoto"/"jpegPhoto" field.
    # "avatar": "thumbnailPhoto",
    # This line is for having Zulip to automatically deactivate users
    # who are disabled in LDAP/Active Directory (and reactivate users who are not).
    # See docs for usage details and precise semantics.
    # "userAccountControl": "userAccountControl",
}
CAMO_URI = '/external_content/'
REMOTE_POSTGRES_HOST = 'database'
REMOTE_POSTGRES_SSLMODE = 'prefer'
AUTHENTICATION_BACKENDS = ('zproject.backends.EmailAuthBackend',)
AUTHENTICATION_BACKENDS += ('zproject.backends.ZulipLDAPAuthBackend',)
ALLOWED_HOSTS = ['chat.dev.mta4.ru', '127.0.0.1']
AUTH_LDAP_BIND_DN = 'cn=s-zulip,ou=zulip,dc=dev,dc=mta4,dc=ru'
AUTH_LDAP_SERVER_URI = 'ldap://dc1.dev.mta4.ru'
AUTH_LDAP_USER_ATTR_MAP = {"full_name": "cn", "avatar": "thumbnailPhoto", "userAccountControl": "userAccountControl"}
AUTH_LDAP_USER_SEARCH = LDAPSearch("ou=zulip,dc=dev,dc=mta4,dc=ru", ldap.SCOPE_SUBTREE, "(mail=%(user)s)")
EMAIL_GATEWAY_IMAP_FOLDER = 'INBOX'
EMAIL_GATEWAY_IMAP_PORT = 143
EMAIL_GATEWAY_IMAP_SERVER = 'mail.dev.mta4.ru'
EMAIL_GATEWAY_LOGIN = 'chat-gkh-streams@dev.mta4.ru'
EMAIL_GATEWAY_PATTERN = 'chat-gkh+%s@dev.mta4.ru'
EMAIL_HOST = 'mail.dev.mta4.ru'
EMAIL_HOST_USER = 'chat-gkh-noreply@dev.mta4.ru'
EMAIL_PORT = 25
EMAIL_USE_SSL = False
EMAIL_USE_TLS = False
ENABLE_FEEDBACK = False
ENABLE_FILE_LINKS = False
ENABLE_GRAVATAR = False
EXTERNAL_HOST = 'chat.dev.mta4.ru'
FEEDBACK_EMAIL = 'ZULIP_ADMINISTRATOR'
INLINE_IMAGE_PREVIEW = True
INLINE_URL_EMBED_PREVIEW = False
JITSI_SERVER_URL = 'meet.jit.si'
LDAP_APPEND_DOMAIN = None
LDAP_EMAIL_ATTR = None
LOCAL_UPLOADS_DIR = '/home/zulip/uploads'
MAX_FILE_UPLOAD_SIZE = 50
MEMCACHED_LOCATION = 'memcached:11211'
NAME_CHANGES_DISABLED = False
PASSWORD_MIN_GUESSES = 10000
PASSWORD_MIN_LENGTH = 8
PUSH_NOTIFICATION_BOUNCER_URL = 'https://push.zulipchat.com'
RABBITMQ_HOST = 'rabbitmq'
RABBITMQ_USER = 'zulip'
RATE_LIMITING = True
REDIS_HOST = 'redis'
REDIS_PORT = 6379
SEND_LOGIN_EMAILS = True
SUBMIT_USAGE_STATISTICS = False
THUMBNAIL_IMAGES = True
THUMBOR_URL = 'http://127.0.0.1:9995'
ZULIP_ADMINISTRATOR = 'chat-gkh-support@dev.mta4.ru'
root@zulip:/#
```

**Comment by rlagutinhub** [comment]
for testing i rerun docker-compose file with ```SETTING_MAX_FILE_UPLOAD_SIZE: '1' ```
and now after attaching the file 10mb i get next error:
![123a](https://user-images.githubusercontent.com/30324030/57179678-e9491800-6e88-11e9-8dbf-8eb88f71955b.png)

**Comment by timabbott** [comment]
Thanks for the report @rlagutinhub!

The error message being hardcoded to 25MiB was fixed recently in master; it'll be in the 2.1 release.  The check was against the correct file size.

I think setting the maximum file upload size much above 25MiB doesn't work, because we don't have a chunked uploader implementation.  There's a separate issue for that in the main project: https://github.com/zulip/zulip/issues/9391

It sounds like the setting is working, and you're just running into the two problems mentioned above, so I'm closing this issue for the Docker project.
